src/templates/users/userprofile_detail.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% load static %}
+
+{% block title %}
+  User:
+  {{ object.dispaly_name }}
+{% endblock title %}
+{% block content %}
+  <div class="container mt-5">
+    <div class="row">
+      <div class="col-md-6 offset-md-3">
+        <div class="card">
+          <div class="card-header">{{ object.display_name }}</div>
+          <div class="card-body">
+            <div class="media">
+              <div class="media-body">
+                <p>Bio: {{ object.bio }}</p>
+                {% if object.user == request.user %}
+                  <a href="{% url 'users:update' %}" class="btn btn-primary">Edit Profile</a>
```

Comment:
Just a quick question. How is this view different from templates/users/user_detail.html file as it also has a similar format to this HTML file?

Reply:
Specifically lines 16-28

Reply:
This is a different view to display a profile page not the user itself. Later, it will include friends, achievement, games, etc. user_detail.html is accessible through the User model unlike userprofile_detail.html which is accessible through UserProfile mode

Reply:
Oh I see, thank you for clarifying, that makes more sense, and it's a great base to build off of.